<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>721</width>
    <height>442</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>721</width>
      <height>391</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tabSports">
     <attribute name="title">
      <string>运动</string>
     </attribute>
     <widget class="QTableWidget" name="tableWidgetSports">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>30</y>
        <width>651</width>
        <height>301</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tabMusic">
     <attribute name="title">
      <string>音乐</string>
     </attribute>
     <widget class="QCustomPlot" name="qCustomPlotMusic" native="true">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>671</width>
        <height>321</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tabSoftware">
     <attribute name="title">
      <string>软件</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>291</width>
        <height>161</height>
       </rect>
      </property>
      <property name="title">
       <string>学习软件</string>
      </property>
      <widget class="QRadioButton" name="radioButton_3">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>96</y>
         <width>91</width>
         <height>18</height>
        </rect>
       </property>
       <property name="text">
        <string>扇贝打卡</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="radioButton_4">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>46</y>
         <width>91</width>
         <height>18</height>
        </rect>
       </property>
       <property name="text">
        <string>Mooc大学</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="radioButton_5">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>50</y>
         <width>91</width>
         <height>18</height>
        </rect>
       </property>
       <property name="text">
        <string>沪江网校</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="radioButton_6">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>100</y>
         <width>91</width>
         <height>18</height>
        </rect>
       </property>
       <property name="text">
        <string>网易公开课</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>10</y>
        <width>291</width>
        <height>161</height>
       </rect>
      </property>
      <property name="title">
       <string>音乐软件</string>
      </property>
      <widget class="QCheckBox" name="checkBox">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>40</y>
         <width>91</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>网易云音乐</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBox_2">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>100</y>
         <width>73</width>
         <height>18</height>
        </rect>
       </property>
       <property name="text">
        <string>酷狗音乐</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBox_3">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>40</y>
         <width>73</width>
         <height>18</height>
        </rect>
       </property>
       <property name="text">
        <string>QQ音乐</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBox_4">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>100</y>
         <width>73</width>
         <height>18</height>
        </rect>
       </property>
       <property name="text">
        <string>虾米音乐</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_5">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>190</y>
        <width>291</width>
        <height>161</height>
       </rect>
      </property>
      <property name="title">
       <string>游戏软件</string>
      </property>
      <widget class="QGroupBox" name="groupBox_6">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>160</y>
         <width>291</width>
         <height>161</height>
        </rect>
       </property>
       <property name="title">
        <string>GroupBox</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>60</y>
         <width>80</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>英雄联盟</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_2">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>110</y>
         <width>80</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>逆战</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_3">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>60</y>
         <width>80</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>CF</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_4">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>110</y>
         <width>80</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>绝地求生</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_7">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>190</y>
        <width>291</width>
        <height>161</height>
       </rect>
      </property>
      <property name="title">
       <string>直播软件</string>
      </property>
      <widget class="QGroupBox" name="groupBox_8">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>160</y>
         <width>291</width>
         <height>161</height>
        </rect>
       </property>
       <property name="title">
        <string>GroupBox</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>50</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>斗鱼直播</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_2">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>50</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>虎牙直播</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_3">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>110</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>熊猫直播</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_4">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>110</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>YY直播</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tabDigital">
     <attribute name="title">
      <string>数码</string>
     </attribute>
     <widget class="QTreeWidget" name="treeWidgetDigital">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>671</width>
        <height>341</height>
       </rect>
      </property>
      <column>
       <property name="text">
        <string notr="true">1</string>
       </property>
      </column>
     </widget>
    </widget>
    <widget class="QWidget" name="tabLanguage">
     <attribute name="title">
      <string>语言</string>
     </attribute>
     <widget class="QListWidget" name="listWidgetLanguage">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>671</width>
        <height>321</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>721</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header location="global">qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
